src/app/shared/issue/description/description.component.ts

```diff
@@ -30,6 +30,11 @@ export class DescriptionComponent implements OnInit {
   @Output() issueUpdated = new EventEmitter<Issue>();
   @Output() changeEditState = new EventEmitter<boolean>();
 
+  // prevents setting of title to bind to defualt html attribute of title
+  @HostBinding('attr.title') get setTitle() {
+    return null;
+  }
+
```

Comment:
this is one way, but I think it makes it harder for future devs to understand what this chunk of code is for.

the more straightforward fix is to rename `title` in the template to something unused like `_title`